src/BoxAnnotations.js

```diff
@@ -49,10 +49,18 @@ class BoxAnnotations {
     /**
      * [constructor]
      *
+     * @param {Object} options - Viewer-specific annotator options
      * @return {BoxAnnotations} BoxAnnotations instance
      */
-    constructor() {
+    constructor(options = {}) {
         this.annotators = ANNOTATORS;
+        this.options = options;
+
+        // Filter out supported annotation types based on provided options
+        Object.keys(options).forEach((annotatorName) => {
+            const annotator = this.annotators[annotatorName];
+            annotator.TYPE = this.getAnnotatorTypes(annotator);
```

Comment:
Guard for whether or not `annotator` exists

Reply:
re-working how these configs are set